client/src/components/categorical/value/index.js

```diff
@@ -439,7 +439,9 @@ class CategoryValue extends React.Component {
 
     if (
       !this.shouldRenderStackedBarOrHistogram ||
-      !AnnotationsHelpers.isContinuousAnnotation(schema, colorAccessor)
+      // This function returns true on categorical annotations(when stacked bar should not render),
+      //  in cases where the colorAccessor is a gene this function will return undefined since genes do not live on the schema
+      AnnotationsHelpers.isCategoricalAnnotation(schema, colorAccessor) === true
```

Comment:
If `isCategoricalAnnotations` returns `undefined` then we _do_ want to render right? In which case wouldn't undefined not equal true thus not rendering?

Reply:
Ahh nvm. I see. `return null` means that it doesn't get rendered and since the conditional will evaluate to false, we'll be clear.

Reply:
If the return value of `isCategoricalAnnotations` is strictly equal to `true` this render function returns null not rendering. `undefined !== true` and doesn't enter the if block

Reply:
Would the alternative of `isContinousAnnotations === false` be more clear?

Reply:
Nah, I'm good! Just took me a second to parse it.